man/systemd-cryptenroll.xml

```diff
@@ -411,6 +411,26 @@
         <xi:include href="version-info.xml" xpointer="v248"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--tpm2-handle=</option><replaceable>HANDLE</replaceable></term>
```

Comment:
can we call this `--tpm2-storage-key=` maybe? i.e. "handle" is just so generic, I think it would be better to say that we reference a *key* here, and what kind of key it is. The handle after all is not the primary concept we want to configure here, but the the object the handle refers to...

other ideas: `--tpm2-parent-key=` or `--tpm2-primary-key=` or `--tpm2-root-key=` or `--tpm2-wrap-key=` or `--tpm2-seal-key=` something like that.

Reply:
so i read from the other PR that this is supposed to deal with a numeric handler as opposed to a key file, hence maybe call this `--tpm2-seal-key-handle=` and `--tpm2-seal-key-file=`?

Reply:
sounds good, updated to `--tpm2-seal-key-handle=`